src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberDecimalDigits.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class NumberFormatInfoNumberDecimalDigits
+    {
+        // PosTest1: Verify property NumberDecimalDigits default value
+        [Fact]
+        public void PosTest1()
```

Comment:
I'm assuming all of these "PosTest" names are because these tests were pre-existing and just imported rather than us choosing these names from scratch?  At some point it'd be nice to update them.